draft-ietf-bidding-and-auction-services.md

```diff
@@ -733,10 +733,22 @@ response = {
   ;   5.       add a tuple to the list in (1) of [interest group owner (2),
                interest group name (4)]
   ;   6. return the list in (1)
-   ? biddingGroups: {
+  ? biddingGroups: {
     * interestGroupOwner => [* int]
   },
 
+  ; Indices and update-if-older-than times of interest groups in the original
+  ; request for this owner for interest groups where an update-if-older-than
+  ; time was specified.
+  ; Maps to https://wicg.github.io/turtledove/#server-auction-response-update-groups.
+  ? updateGroups: {
+    * interestGroupOwner => [
+      * {
+        index: int,
+        time: int

```

Comment:
I've been calling this updateIfOlderThanMs in our code - if we want to rename that's ok, but i'll need to patch. Also somewhere we should specify this int represents Ms

Reply:
Odd. I could have sworn I was working from the design doc, but I couldn't find a reference to "time" anywhere. It's not really a time (since it's a duration) so I'm open to changing. This feature hasn't launched yet, so I can just put together a fix. That being said, we should try to keep the field names short since they are encoded in CBOR messages.